extensions/CheckUser/CheckUser.hooks.php

```diff
@@ -346,16 +355,16 @@ public static function doRetroactiveAutoblock( Block $block, array &$blockIds )
 
 		$user = User::newFromName( (string)$block->getTarget(), false );
 		if ( !$user->getId() ) {
-			return array(); // user in an IP?
+			return []; // user in an IP?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)